packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+    timeout: {
+      type: Number,
+      default: 8000,
+    },
+    closeCallBack: {
```

Comment:
Can you please use "onClose" instead of "closeCallback" as we did in `VtmnToast` react component?